Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2144 @@
+    return raw_response
+
+
+def get_incident_uuid(endpoint: str, client: Client, args: Dict[str, Any]) -> str:
+    """
+      Get the incident UUID
+      Args:
+          endpoint: API endpoint
+          client: client object to use.
+          args: all command arguments, usually passed from ``demisto.args()``.
+      Returns:
+          CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains an updated
+              result.
+    """
+    # Get UUID based on incident_id
+    incident_uuid = None
+    data_list = get_incident_raw_response_data(endpoint, client, args, 1).get('result', [])
+    if len(data_list) >= 1:
+        incident_uuid = data_list[0].get('uuid')
+    else:
+        raise DemistoException(f'Incident ID Not Found {args.get("incident_id")}.'
+                               f'Provide time range arguments if incidents is older then 30 days')
+
+    return incident_uuid
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """
+    Tests API connectivity and authentication
+    When 'ok' is returned it indicates the integration works like
+    it is supposed to and connection to the service is successful.
+    Args:
+        client(Client): Client class object
+    Returns:
+        Connection ok
+    """
+    message: str = ''
+    endpoint = '/atpapi/v2/incidents'
+    params: Dict = {
+        'verb': "query",
+        'limit': 1
+    }
+    try:
+        client.query_request_api(endpoint, params)
+        message = 'ok'
+
+    except DemistoException as e:
+        if 'Authorization Error' in str(e) or 'Unauthorized' in str(e):
+            message = f'Make sure Client ID and Client Secret are correctly set. {e}'
+        elif 'Timeout Error' in str(e):
+            message = f'Server URL Invalid. {e}'
+        else:
+            message = f'Either Client ID or Client Secret is Invalid. {e}'
+        raise DemistoException(message)
+
+    return message
+
+
+def get_domain_file_association_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    List of Domain and File association
+    Args:
+        client: Symantec EDR on-premise client objectd to use.
+        args: all command arguments, usually passed from ``demisto.args()``.
+    Returns:
+        CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains an updated
+            result.
+    """
+    endpoint = '/atpapi/v2/associations/entities/domains-files'
+
+    page = arg_to_number(args.get('page', 1), arg_name='page')
+    page_size = arg_to_number(args.get('page_size', DEFAULT_PAGE_SIZE), arg_name='page_size')
+    page_limit, offset = pagination(page, page_size)
+    payload = post_request_body(args, page * page_limit)
+    # search query as Lucene query string
+    search_query = get_association_filter_query(args)
+    if search_query:
+        payload['query'] = search_query
```

Comment:
these lines are repeated many times, please replace them with a function.

Reply:
Created generic function call "get_request_payload" which will return payload, and paginations values